open_ce/build_feedstock.py

```diff
@@ -104,6 +104,9 @@ def build_feedstock_from_command(command, # pylint: disable=too-many-arguments,
     '''
     utils.check_if_conda_build_exists()
 
+    if 'cuda' in command.build_type:
+        utils.check_cuda_version_match(command)
```

Comment:
Do we need to stop build here, with an error in case the version does not match?

Reply:
That is what I did in this PR originally (I made it a fatal error).  However, this caused the CI build to fail to complete,
because during the CI, three factors combine to make it fail there: (1) CI includes cuda build, (2) CUDA_HOME isn't pre-specified in CI (so this code sets it to default /usr/local/cuda), but (3) the /usr/local/cuda directory isn't present in the CI build -- so if this is a "fatal" error then the CI stops.
Thus, I changed the function to print a WARNING in this situation rather than fatally end the build.  This provides a check and notification that does not exist today (before this PR exists).  Maybe if there's a way to check explicitly that
this is a CI build, I could do that; otherwise making this just a non-fatal warning wouldn't kill the build but would still
serve a notice useful to someone building outside of the CI (if they are watching for warnings).

Reply:
I see.  Yes, so I think for the case when  `/usr/local/cuda/version.txt`  isn't present WARNING is good. 

However, when `/usr/local/cuda/version.txt` is present, but versions do not match ,the build will be successful without any message to the user. Do we want to give a WARNING/Error in this case ?

Reply:
@cdeepali Yes, that was the intent; an earlier iteration did so as I remember testing it, but I somehow lost this case along the way.  I've now added a warning message specific for the mismatch case -- not here, but in `utils.py` when returning.  (A T/F return value is provided by the utility function for possible use, but is not actually used here.)